### PR TITLE
Release v3.4 and v2.17 post and config

### PR DESCRIPTION
**Pull request by mattfarina**
This includes two things...
1. A blog post on 3.4 and 2.17 that talks about the charts move
2. param updates for the new versions

### CODE REVIEW

File: content/en/blog/2020-10-26-new-location-stable-incubator-charts.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Helm v2.17.0
+
+Helm v2 added the stable repository by default when `helm init` was run. This has led to a different solution for Helm v2, starting in v2.17.0.
+
+If you do not need the stable or local repositories, you can use the `--skip-repos` flag when running `helm init`. This is a new flag in v2.17.0. This can have some performance benefits in some use cases such as CI systems where you aren't using the stable repository.
+
+In v2.17.0, when `helm init` is run the new location is used instead instead of the old location. This is what will happen in CI systems that regularly run `helm init`. If you need to continue to use the old location, you can pass the new `--use-deprecated-stable-repository` flag to `helm init`. This will only work for as long as the old locations continue to operate.
```
Comment (flynnduism) [inline content/en/blog/2020-10-26-new-location-stable-incubator-charts.md:42]: > the new location is used **instead instead** of the

Duplication of _instead_ here

**Review by bacongobbler — APPROVED** [review]
content LGTM

**Review by flynnduism — CHANGES_REQUESTED** [review]
One small typo in the blog post, once you fix the ~~instead~~ it's a LGTM

**Review by flynnduism — APPROVED** [review]
LGTM

**Comment by bridgetkromhout** [comment]
Deploy preview looks good.

**Pull request merged**